README.md

```diff
@@ -27,3 +28,7 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+Answer:
+- At the beginning of every Nth recursion there is an array of N+1 integers where N $\ge$ 0: fibArray[0] = 0, fibArray[1] = 1, fibArray[2] = 1
+- At the beginning of every Nth recursion where n > 2: fibArray[n] is equal to fibArray[n-1] + fibArray[n-2]
```

Comment:
But that's not true. You're calculating the $n$th number, it's not available before you do that.